tests/container-scripts/run-testnet.sh

```diff
@@ -21,7 +21,7 @@ do
         RPC_ADDRESS="--rpc.laddr tcp://0.0.0.0:26657"
         GRPC_ADDRESS="--grpc.address 0.0.0.0:9090"
         GRPC_WEB_ADDRESS="--grpc-web.address 0.0.0.0:9092"
-        ETH_RPC_ADDRESS="--json-rpc.address 0.0.0.0:8545"
```

Comment:
This config allows the ETH JSON-RPC to start but it is still inaccessible from outside the container, which will hamper testing and providing public testnet endpoints in the future